autogluon/task/tabular_prediction/predictor.py

```diff
@@ -688,6 +688,75 @@ def load_data_internal(self, dataset='train', return_X=True, return_y=True):
         y = load_y() if return_y else None
         return X, y
 
+    def reduce_memory_size(self, remove_data=True, remove_fit_stack=True, requires_save=True, reduce_children=False):
+        """
+        Reduces the memory and disk size of `predictor`.
```

Comment:
may want to clarify: 

"Reduces the memory and disk size of `predictor` by deleting auxiliary model files that aren't needed for prediction on new data.  We recommend invoking this method if your only goal is to use the trained model for prediction.  However, certain advanced functionality may no longer be available after `reduce_memory_size()` has been called."

Reply:
Added